src/scheduler.rs

```diff
@@ -0,0 +1,14 @@
+use tokio_cron_scheduler::{Job, JobScheduler, JobSchedulerError};
+
+pub async fn run_scheduler() -> Result<(), JobSchedulerError> {
+    let scheduler = JobScheduler::new().await?;
+
+    let checker_job = Job::new("1/10 * * * * *", |_uuid, _l| {
```

Comment:
what does `1/10 * * * * *` mean?

Reply:
good question, just copied this from their docs. It's once every 10 seconds

Reply:
ah thanks, sorry wasn't familiar with that format